        -:    0:Source:/home/wpsimon09/Desktop/LearningVulkan/Vulkan/src/main.cpp
        -:    0:Graph:/home/wpsimon09/Desktop/LearningVulkan/Vulkan/cmake-build-debug-coverage/CMakeFiles/LearnVulkan.dir/src/main.cpp.gcno
        -:    0:Data:/home/wpsimon09/Desktop/LearningVulkan/Vulkan/cmake-build-debug-coverage/CMakeFiles/LearnVulkan.dir/src/main.cpp.gcda
        -:    0:Runs:1
        -:    1:#include <iostream>
        -:    2:
        -:    3:#include "VulkanApp.hpp"
function main called 1 returned 100% blocks executed 41%
        1:    4:int main() {
        -:    5:
        1:    6:    VulkanApp app;
        1:    6-block 2
call    0 returned 1
        -:    7:
        -:    8:    try {
        1:    9:        app.run();
call    0 returned 1
branch  1 taken 1 (fallthrough)
branch  2 taken 0 (throw)
    =====:   10:    }catch(const std::exception &e) {
    $$$$$:   10-block 8
branch  0 never executed (fallthrough)
branch  1 never executed
    $$$$$:   10-block 10
call    2 never executed
    =====:   11:        std::cerr<<e.what();
call    0 never executed
call    1 never executed
branch  2 never executed (fallthrough)
branch  3 never executed (throw)
    =====:   12:        return EXIT_FAILURE;
    =====:   13:    }
    $$$$$:   13-block 13
call    0 never executed
    $$$$$:   13-block 14
call    1 never executed
        -:   14:
        1:   15:    return EXIT_SUCCESS;
        1:   15-block 4
        1:   16:}
        1:   16-block 5
call    0 returned 1
    $$$$$:   16-block 16
call    1 never executed
